Source/RxSwift/ObservableType+Weak.swift

```diff
@@ -83,6 +119,33 @@ extension ObservableType {
 
 		return Disposables.create(self.asObservable().subscribe(observer), disposable)
 	}
+    
+    /**
+     Subscribes an element handler, an error handler, a completion handler and disposed handler to an observable sequence.
+     
+     - parameter weak: The object that we want to have a safe and unretained reference on.
+     - parameter onNext: Action to invoke for each element in the observable sequence.
+     - parameter onError: Action to invoke upon errored termination of the observable sequence..
+     - parameter onCompleted: Action to invoke upon graceful termination of the observable sequence..
+     - parameter onDisposed: FAction to invoke upon any type of termination of sequence (if the sequence has gracefully completed, errored, or if the generation is canceled by disposing subscription).
```

Comment:
`FAction -> Action` , but also in general, can we use the same docs from the original `subscribe(weak)` ? It uses wording like function, vs. action

Reply:
Done